dashboard/admin.py

```diff
@@ -70,9 +70,15 @@ def has_add_permission(self, request):
 class CourseAdmin(admin.ModelAdmin):
     inlines = [CourseViewOptionInline, ]
     form = CourseForm
-    list_display = ('canvas_id', 'name', 'term', 'show_grade_counts', 'course_link', '_courseviewoption', 'show_grade_type')
```

Comment:
We just removing `show_grade_type` because it was never implemented?

Reply:
Since we still have the 'show_grade_type' column in course table, I suggest we still show this value.

Reply:
Right, I switched `show_grade_type` to `data_last_updated` based on a suggestion from the meeting. `show_grade_type` will always be the same and currently doesn't do anything.

I can re-add it or not make any change here but the table has a lot of columns and we should only show the most necessary information here on the table.

Reply:
Given the concern of the table space, I am fine either way of adding back the show_grade_type or not.